### PR TITLE
Update all 

### PR DESCRIPTION
**Pull request by ZeusJunior**
Add ability to use !update all=true with intent/min/max/enabled/autoprice as optional parameters
#157

### CODE REVIEW

File: app/handler/commands.js [hunk 1]
```diff
@@ -552,6 +552,45 @@ exports.handleMessage = function (steamID, message) {
...
+        if (typeof params.intent === 'string') {
+            const intent = ['buy', 'sell', 'bank'].indexOf(params.intent.toLowerCase());
+            if (intent !== -1) {
+                params.intent = intent;
+            }
+        }
+
+        if (params.all === true) {
+            // TODO: Must have atleast one other param
+            client.chatMessage(steamID, 'Attempting to update all, snoozing...');
```
Comment (Nicklason) [inline app/handler/commands.js]: Jjust have it say "Updating pricelist..."

File: app/handler/commands.js [hunk 2]
```diff
@@ -552,6 +552,45 @@ exports.handleMessage = function (steamID, message) {
...
+                if (params.max && typeof params.max === 'number') {
+                    pricelist[i].max = params.max;
+                }
+                if (params.enabled === false || params.enabled === true) {
+                    pricelist[i].enabled = params.enabled;
+                }
+                if (params.autoprice === false || params.autoprice === true) {
+                    pricelist[i].autoprice = params.autoprice;
+                }
+                pricelist[i].time = 0;
```
Comment (Nicklason) [inline app/handler/commands.js]: Test with the first pricelist entry if it matches the pricelist schema

File: app/handler/commands.js [hunk 3]
```diff
@@ -552,6 +552,45 @@ exports.handleMessage = function (steamID, message) {
...
+            }
+            // Save pricelist
```
Comment (Nicklason) [inline app/handler/commands.js]: Include new line above this

File: app/handler/commands.js [hunk 4]
```diff
@@ -552,6 +552,45 @@ exports.handleMessage = function (steamID, message) {
...
+            handlerManager.getHandler().onPricelist(pricelist);
+            client.chatMessage(steamID, 'Pricelist updated, shutting down...');
```
Comment (Nicklason) [inline app/handler/commands.js]: Remove this message

File: app/handler/commands.js [hunk 5]
```diff
@@ -552,6 +553,71 @@ exports.handleMessage = function (steamID, message) {
...
+
+            if (pricelist.length === 0) {
+                client.chatMessage(steamID, 'Your pricelist is empty');
+                return;
+            }
+
+            handlerManager.getHandler().cleanup();
+
+            for (i = 0; i < pricelist.length; i++) {
+                if (params.intent) {
```
Comment (Nicklason) [inline app/handler/commands.js]: Check if intent is a number

Comment (ZeusJunior) [reply app/handler/commands.js]: No i didn't add that because i moved the typeof for intent up

**Pull request merged**